lightning/src/ln/features.rs

```diff
@@ -66,6 +66,8 @@
 //!      for more info).
 //! - `Keysend` - send funds to a node without an invoice
 //!     (see the [`Keysend` feature assignment proposal](https://github.com/lightning/bolts/issues/605#issuecomment-606679798) for more information).
+//! - `Trampoline` - send funds to a node without without knowing a direct path using nested onions
```

Comment:
Does the feature bit indicate support for forwarding as a trampoline node? This sounds like it's for sending.